instagram/src/components/CommentSection/LikesSection.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+const PostFeedbackContain = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const Far = styled.i`
+    font-size: 30px;
+    font-weight: 100;
+    margin: 5px 10px;
+    -webkit-text-stroke: 1px white;
+`;
+
+const FeedbackItems = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+
+const LikeAmount = styled.div`
+    text-align: left;
+    font-weight: bold;
+    padding-left: 15px;
+`;
+
+class LikesSection extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            likes: props.likes,
+        }
+    };
+
+    incrementLike = () => {
+        let likes = this.state.likes + 1;
```

Comment:
I believe we covered this in our 1:1 - instagram actually has the increment to be a plus and minus 1. If you were to apply this, how would you go about it?? (Does not have to be in detail, more of a psuedocode thing or just a thought!)

Reply:
I'd have to spend a little bit of time on this, but in theory you would simply use an onClick on the heart icon to toggle a boolean value (true or false) that would be added on state. Then use some logic to increment the count (if true, count + 1; if false, do nothing to the count). Essentially something like this. I may return to tinker with it sometime this week.